reference/api/tasks.md

```diff
@@ -117,17 +117,27 @@ This value is always `null` for `dumpCreation` tasks.
 
 #### `taskDeletion`
 
+| Name                  | Description                                                                                                                                                                                                                    |
+| :-------------------- | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **`matchedTasks`**    | The number of matched tasks based on the request. If the API key used with the request doesn’t have access to any of the indexes specified in the request, tasks relating to that index will not be included in `matchedTasks` |
```

Comment:
Ooops, that's a lot of "request"s

```suggestion
| **`matchedTasks`**    | The number of matched tasks. If the API key used with the request doesn’t have access to any of the specified indexes, tasks relating to that index will not be included in `matchedTasks` |
```

Reply:
Thanks! I was supposed to reword this 🤦‍♀️